catalog/spatial/geocoding/spatial-geocoding-admin-module/src/main/webapp/js/HandlebarsHelpers.js

```diff
@@ -0,0 +1,346 @@
+/**
```

Comment:
Seems like a good candidate for a sub project so we can pull this in as a bower dependency instead of copying it all over the place?